src/styles/slide.css

```diff
@@ -32,6 +32,15 @@
         padding-top: 0;
       }
     }
+
+    a {
```

Comment:
@Envek Will be better to use selector by class name instead of element selector. `&--link` for example. And then use it as `a.slide--link.gtm-link` in the markup

Reply:
`&--link` doesn't work because this declaration is nested into `.slide .red` (as it is specific only for “red” slides), so I had to write full `.slide--link` instead. Does it make sense?